### PR TITLE
[doc] PR for Documentation for GCP KMS

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-4000

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    </a>
+  </li>
+
+</ul>
+
+Encryption at rest uses universe keys to encrypt and decrypt universe data keys. You can use the YugabyteDB Anywhere UI to create key management service (KMS) configurations for generating the required universe keys for one or more YugabyteDB universes. Encryption at rest in YugabyteDB Anywhere supports the use of Google Cloud KMS. 
+
+Conceptually, Google Cloud KMS consists of a key ring containing one or more cryptographic keys, with each key capable of having multiple versions.
+
+Before defining a KMS configuration with YugabyteDB Anywhere, you need to create a key ring through [Google Cloud KMS](https://cloud.google.com/security-key-management).
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md:50]: This is actually not necessary. If a key ring with the same name already exists, it validates and uses the existing key ring. Else, a new one is created automatically.

File: docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+1. Use the YugabyteDB Anywhere UI to navigate to **Configs > Security > Encryption At Rest** to access the list of existing configurations.
+
+1. Click **Create New Config**.
+
+3. Enter the following configuration details in the form:
+
+    - **Configuration Name** — Enter a meaningful name for your configuration.
+    - **KMS Provider** — Select **GCP KMS**.
+    - **Service Account Credentials** — Download the JSON file containing your Google Cloud credentials.
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md:81]: Shouldn't this be "Upload the JSON file..."?

**Review by Sahith02 — CHANGES_REQUESTED** [review]

File: docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md:50]: Deleted the "Before..." sentence

File: docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md:81]: Fixed

**Comment by lizayugabyte** [comment]
Done. The PR will be merged for 2.16

**Review by Sahith02 — APPROVED** [review]
LGTM!

File: docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- `cloudkms.keyRings.create`
+- `cloudkms.keyRings.get`
+- `cloudkms.keyRings.list`
+- `cloudkms.cryptoKeys.create`
+- `cloudkms.cryptoKeys.get`
+- `cloudkms.cryptoKeys.list`
+- `cloudkms.cryptoKeyVersions.useToEncrypt`
+- `cloudkms.cryptoKeyVersions.useToDecrypt`
+- `cloudkms.cryptoKeyVersions.destroy`
+- `cloudkms.locations.generateRandomBytes`
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md:61]: @lizayugabyte I've updated the permissions section of the design doc. Can you please take a look at the doc and update this section accordingly.

File: docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+</ul>
+
+Encryption at rest uses universe keys to encrypt and decrypt universe data keys. You can use the YugabyteDB Anywhere UI to create key management service (KMS) configurations for generating the required universe keys for one or more YugabyteDB universes. Encryption at rest in YugabyteDB Anywhere supports the use of Microsoft Azure KMS. 
+
+Conceptually, Azure KMS consists of a key vault containing one or more keys, with each key capable of having multiple versions.
+
+Before defining a KMS configuration with YugabyteDB Anywhere, you need to create a key vault through the [Azure portal](https://docs.microsoft.com/en-us/azure/key-vault/general/quick-create-portal). The following settings are required:
+
+- Set the vault permission model as Vault access policy.
+- Add the application to the key vault access policies with the minimum key management operations permissions of Get, List, Create, and cryptographic operations permissions of Unwrap Key, Wrap Key.
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md:53]: Can we also change this to reflect the new changes in [this PR](https://github.com/yugabyte/yugabyte-db/pull/13916).

**Review by Sahith02 — CHANGES_REQUESTED** [review]

File: docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md:61]: New list:

- `cloudkms.keyRings.create`
- `cloudkms.keyRings.get`
- `cloudkms.cryptoKeys.create`
- `cloudkms.cryptoKeys.get`
- `cloudkms.cryptoKeyVersions.useToEncrypt`
- `cloudkms.cryptoKeyVersions.useToDecrypt`
- `cloudkms.locations.generateRandomBytes`

File: docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md:53]: @Sahith02 I'm not sure what this comment is referring to.

Comment (Sahith02) [reply docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md:53]: @lizayugabyte I mean changing line 53 to what is modified in the [comment here](https://github.com/yugabyte/yugabyte-db/pull/13916#discussion_r976085912).

**Review by Sahith02 — APPROVED** [review]

**Pull request merged**